packages/react/src/components/TimePicker/TimePicker.types.ts

```diff
@@ -60,32 +64,43 @@ export interface ITimePickerProps
   allowFreeform?: boolean;
 
   /**
-   * Custom time range to for time options
+   * Custom time range to for time options.
    */
   timeRange?: ITimeRange;
 
   /**
-   * Localized strings to use in the TimePicker
+   * Localized strings to use in the TimePicker.
    */
   strings?: ITimePickerStrings;
 
   /**
-   * Default value of the TimePicker, if any
+   * The uncontrolled default selected time.
```

Comment:
```suggestion
   * The uncontrolled default selected time.
   * Mutually exclusive with `value`.
```

Reply:
Resolved by updating comment to suggested.